CHANGES.md

```diff
@@ -1,6 +1,8 @@
 # Changes
 
 This document describes the relevant changes between releases of the API model.
+## 0.0.343 December 12 2023
```

Comment:
Should be 0.0.352 now and date should be updated

Reply:
Thank you. :-) Updated.